apps/preview/next/pages/showcases/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,154 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton, SfIconChevronLeft, SfIconChevronRight, usePagination } from '@storefront-ui/react';
+import classNames from 'classnames';
+
+export function Showcase() {
+  const { totalPages, pages, selectedPage, startPage, endPage, next, prev, onPageChange, maxVisiblePages } =
+    usePagination(150, 2, 6, 4);
```

Comment:
Right now by looking on `hook`/`composable` i have no idea what each parameter do, what do you think about pass data as object ? 

```suggestion
    usePagination({
       toalItems: 150,
       currentPage: 2,
       pageSize: 10,
       maxPages: 1
    });
```

Reply:
sure, looks better :)

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2535/commits/3ac7e43a8ab6b69cd07bd650bc5f0163b76f477b